projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -71,7 +77,16 @@ export class TabParagraphContainerComponent
   );
 
   select(tabNum: number): void {
-    this.activeTabNum = tabNum;
+    this.breakpointService.isDown(BREAKPOINT.sm).subscribe((res) => {
+      if (res) {
+        this.activeTabNum = this.activeTabNum === tabNum ? -1 : tabNum;
+        let targets = this.target.nativeElement.getBoundingClientRect();
+        const top = targets.top + window.scrollY || window.pageYOffset;
+        top > 1220 ? window.scrollTo(0, 1214.5) : window.scrollTo(0, top);
```

Comment:
Hey Temi,

Can you explain this solution? Why you rely on such static numbers like: `1220`  and `1214.5`?